test/textfield.test.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { mount } from 'enzyme'
+import { TextField } from '../src/components/TextField'
+
+describe('TextField', () => {
```

Comment:
I think that a test to make sure that the `onChange` handler gets passed in and is working properly would be helpful!

We did something similar for `onClick` in `Button` for reference.

Reply:
This is now resolved.